static/compiler.js

```diff
@@ -504,13 +504,12 @@ define(function (require) {
     };
 
     Compiler.prototype.onCompilerSetDecorations = function (id, lineNums) {
-        if (id === this.id) {
+        if (id == this.id) {
```

Comment:
Why so? It seems that `===` should be the right thing to do here? can you get it to print the type of `id` and `this.id` - they ought both to be integers...but I suspect one is a string somehow...oh javascript...